doc/AOM_ANALYZER.md

```diff
@@ -0,0 +1,25 @@
+# AOMAnalyzer
+
+<details>
+<summary><b>Table of Content</b></summary>
+
+- [Local Analyzer](#local-analyzer)
+- [Online Analyzer](#online-analyzer)
+</details>
+
+## Local Analyzer
+
+1. Download the [AOM Analyzer](http://aomanalyzer.org) ([source](https://github.com/xiph/aomanalyzer)).
+2. Download [inspect.js](https://people.xiph.org/~mbebenita/analyzer/inspect.js) and [inspect.wasm](https://people.xiph.org/~mbebenita/analyzer/inspect.wasm) and save them in the same directory.
```

Comment:
Those files aren't available, please point to the correct place (@tdaede which is it?)

Reply:
Found them: https://github.com/xiph/aomanalyzer/issues/80